### PR TITLE
APPSRE-4404 schema for githubRepoInvites

### PR DESCRIPTION
**Pull request by fishi0x01**
**Motivation**

We currently store configuration for the `github-repo-invites` integration in our `config.toml`. This PR adds a schema to move the `[github-repo-invites]` toml section to our `app-interface` repo.

**Test**

Successfully ran:

```
make bundle
```

Further, tested with `--dry-run` in corresponding [q-r PR](https://github.com/app-sre/qontract-reconcile/pull/2239) in qontract-reconcile

### CODE REVIEW

File: schemas/app-interface/app-interface-settings-1.yml [hunk 1]
```diff
@@ -48,6 +48,12 @@ properties:
         type: string
       credentials:
         "$ref": "/common-1.json#/definitions/vaultSecret"
+  githubRepoInvites:
+    type: object
```
Comment (rporres) [inline schemas/app-interface/app-interface-settings-1.yml:52]: Please add descriptions for the new fields.

File: schemas/app-interface/app-interface-settings-1.yml [hunk 2]
```diff
@@ -48,6 +48,12 @@ properties:
...
+    additionalProperties: false
+    properties:
+      credentials:
+        "$ref": "/common-1.json#/definitions/vaultSecret"
```
Comment (rporres) [inline schemas/app-interface/app-interface-settings-1.yml:56]: I guess you want to make `credentials` required, right?

**Review by rporres — COMMENTED** [review]
app-interface settings is becoming a mixed bag of too many loosely related things, but I don't think we have another place for this kind of information

File: schemas/app-interface/app-interface-settings-1.yml (hunk 1 above)
Comment (fishi0x01) [reply schemas/app-interface/app-interface-settings-1.yml:52]: Fixed

Comment (rporres) [reply schemas/app-interface/app-interface-settings-1.yml:52]: No, not that way 😄  A description field that documents the new field, e.g. https://github.com/app-sre/qontract-schemas/blob/main/schemas/app-sre/slo-document-1.yml#L42

Comment (fishi0x01) [reply schemas/app-interface/app-interface-settings-1.yml:52]: I see. Adjusted it now :)

**Review by rporres — APPROVED** [review]

**Pull request merged**